sdk/analytics/src/main/java/com/klaviyo/analytics/model/Event.kt

```diff
@@ -5,18 +5,24 @@ import java.io.Serializable
 /**
  * Controls the data that can be input into a map of event attributes recognised by Klaviyo
  */
-class Event(val type: EventType, properties: Map<EventKey, Serializable>?) :
```

Comment:
Is this public should you have a deprecated version of this?

Reply:
Oh good idea. I did try that out just now but I can't get that to compile. However, this signature change is not breaking, since EventMetric is a superclass of EventType. It just means their code will only show a deprecated warning on the EventType constructor, but not for the Event constructor.